sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/keyvault/KeyVaultEnvironmentPostProcessorHelper.java

```diff
@@ -131,12 +132,12 @@ public TokenCredential getCredentials() {
      */
     public TokenCredential getCredentials(String normalizedName) {
         //use service principle to authenticate
-        final String clientId = getPropertyValue(normalizedName, Property.CLIENT_ID);
+        final String clientId = getProperty(normalizedName, Property.CLIENT_ID);
         final String clientKey = getPropertyValue(normalizedName, Property.CLIENT_KEY);
```

Comment:
The embrassing thing is keyvault uses client-key instead of client-secret, so do you think we should read value of azure.keyvault.client-key or sping.cloud.azure.client-secret?